kit/src/main/java/com/oracle/javafx/scenebuilder/kit/skeleton/SkeletonWindowController.java

```diff
@@ -120,6 +126,28 @@ public SkeletonWindowController(EditorController editorController, String docume
         if (editorController.getFxomDocument() != null) {
             editorController.getFxomDocument().sceneGraphRevisionProperty().addListener(fxomDocumentRevisionListener);
         }
+        
+        this.copyAccelerator = getCopyToClipboardKeyAccelerator();
+    }
+
+    private KeyCharacterCombination getCopyToClipboardKeyAccelerator() {
+        return new KeyCharacterCombination("c", KeyCombination.SHORTCUT_DOWN);
+    }
```

Comment:
@abhinayagarwal We now can even consider to remove this method at all. No need to have a method now as the conditional is gone. Still need to test on Mac. Works already fine on Windows 10.

Reply:
Tested on mac. Working fine.

Reply:
Great to hear! My Mac is not ready yet, so seems fine to go.